pkg/vm/precompiles.go

```diff
@@ -15,69 +15,32 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-func contains(list []common.Address, element common.Address) bool {
-	for _, val := range list {
-		if val == element {
-			return true
-		}
-	}
-	return false
-}
-
 func getAdminList(initialPrompt string, info string, app *application.Avalanche) ([]common.Address, bool, error) {
-	const (
-		addAdmin    = "Add admin"
-		removeAdmin = "Remove admin"
-		preview     = "Preview"
-		moreInfo    = "More info"
-		doneMsg     = "Done"
-		cancelMsg   = "Cancel"
+	label := "Address"
+
+	list, canceled, err := app.Prompt.CaptureListDecision(
+		app.Prompt,
```

Comment:
improve params description, or naming

Reply:
Done 3 times in interface spec, in implementation and first use. I think that's sufficient.